index.html

```diff
@@ -5154,7 +5154,56 @@ <h2>The <code>application/credential+ld+json</code> Media Type</h2>
   </p>
 
 </section>
+<section id="verifiable-credential-media-type">
+  <h2>The <code>application/verifiable-credential+ld+json</code> Media Type</h2>
+  <p>
+    This specification registers the <code>application/verifiable-credential+ld+json</code> MIME Media Type specifically for
+    identifying documents
+    conforming to the Verifiable Credentials format.
+  </p>
+  <table>
+    <tr>
+      <td>Type name: </td>
+      <td>application</td>
+    </tr>
+    <tr>
+      <td>Subtype name: </td>
+      <td>verifiable-credential+ld+json</td>
+    </tr>
+    <tr>
+      <td>Required parameters: </td>
+      <td>None</td>
+    </tr>
+    <tr>
+      <td>Encoding considerations: </td>
+      <td>
+        Resources that use the "<code>application/verifiable-credential+ld+json</code>"
+        Media Type are required to conform to all of the requirements
+        for the "<code>application/credential+ld+json</code>" Media Type and are
+        therefore subject to the same encoding considerations specified
+        in Section 11 of [[!RFC7159]].
+      </td>
+    </tr>
+    <tr>
+      <td>Security considerations: </td>
+      <td>As defined in this specification.</td>
```

Comment:
```suggestion
      <td>As defined in the Verifiable Credentials Data Model <a href="https://w3c.github.io/vc-data-model/#security-considerations">Security Considerations</a> section.</td>
```

This section is supposed to be used as input into the IANA registry and it wouldn't make much sense for the IANA registry to say "As defined in this specification."

I'll also note that all of the current registrations in the spec (and the PRs) are incomplete. They need to specify all of the items in the IANA registration template here: https://www.rfc-editor.org/rfc/rfc6838.html#section-5.6

So, we'll need future PRs to clean all of that up, AFAICT. That doesn't need to hold up this PR, just noting that we need to remember to make sure we do this in the future.